web/css/checkboxes.css

```diff
@@ -76,7 +76,6 @@ label.switch span {
   color: #95a4ab;
   text-align: left;
   -webkit-user-select: none;
-     -moz-user-select: none;
       -ms-user-select: none;
           user-select: none;
```

Comment:
are we not supporting mozilla

Reply:
Yes we are. tbh I dont know why this was removed / what removed it...ill try to restore it